components/login/provider/index.jsx

```diff
@@ -33,217 +33,203 @@ import {
                     <img
                       src={`../${option.logo}`}
                       width={20}
-                      style={{ margin: "1rem" }}
                       alt={option.label}
                     />
-                  ) : (
-                    <div style={{ minWidth: "20px", padding: "1.6rem" }} />
                   )}
-                  {option.label}
+                  <div
+                    style={{
+                      marginLeft: option.logo ? "0px" : "20px",
+                      padding: "0.5rem 1rem",
+                    }}
+                  >
+                    {option.label}
+                  </div>
                 </>
               );
             }}
             renderInput={(params) => {
               return (
                 <TextField
-                  onChange={handleChange}
                   {...params}
+                  inputProps={{
+                    ...params.inputProps,
+                    autoComplete: "off",
+                    inputMode: "url",
+                  }}
                   error={!!loginError}
                   margin="none"
                   variant="outlined"
-                  type="url"
                   aria-describedby={loginError ? "login-error-text" : null}
                   data-testid={TESTCAFE_ID_LOGIN_FIELD}
                   ref={loginFieldRef}
                   value={inputValue}
-                  onKeyUp={(e) => {
-                    if (e.key === "Enter") {
-                      handleLogin(e, providerIri);
-                    }
-                  }}
                 />
               );
             }}
           />
           {loginError ? (
-            <FormHelperText
-              id="login-error-text"
-              style={{ marginBottom: theme.spacing(1) }}
-            >
-              {getErrorMessage(loginError)}
+            <FormHelperText id="login-error-text" style={{ marginBottom: 0 }}>
+              {loginError}
             </FormHelperText>
-          ) : null}
+          ) : (
+            <p style={{ margin: 0 }}>&nbsp;</p>
+          )}
         </FormControl>
-        <LoginButton
-          oidcIssuer={providerIri}
-          redirectUrl={generateRedirectUrl("")}
-          authOptions={AUTH_OPTIONS}
-          onError={onError}
+        <Button
+          type="submit"
+          variant="primary"
+          className={bem("provider-login__button")}
+          data-testid={TESTCAFE_ID_GO_BUTTON}
         >
-          <Button
-            variant="primary"
-            data-testid={TESTCAFE_ID_GO_BUTTON}
-            type="submit"
-            className={bem("provider-login__button")}
-            onClick={() => setProviderIri(inputValue)}
-          >
-            Go
-          </Button>
-        </LoginButton>
+          Go
+        </Button>
       </div>
     </form>
   );
 }
 
 Provider.propTypes = {
-  // eslint-disable-next-line react/forbid-prop-types
-  defaultError: T.object,
```

Comment:
This seemed to just exist for testing, as it was never passed when the component was actually used in the app — We can work on a different testing strategy for this instead.